lib/WorseReflection/Bridge/Phpactor/DocblockParser/PHPStanDocblockParserFactory.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Phpactor\WorseReflection\Bridge\Phpactor\DocblockParser;
+
+use Phpactor\WorseReflection\Core\DocBlock\PlainDocblock;
+use Phpactor\WorseReflection\Core\DocBlock\DocBlock;
+use Phpactor\WorseReflection\Core\DocBlock\DocBlockFactory;
+use Phpactor\WorseReflection\Core\Reflection\ReflectionScope;
+use Phpactor\WorseReflection\Reflector;
+use PHPStan\PhpDocParser\Lexer\Lexer;
+use PHPStan\PhpDocParser\Parser\ConstExprParser;
+use PHPStan\PhpDocParser\Parser\PhpDocParser;
+use PHPStan\PhpDocParser\Parser\TokenIterator;
+use PHPStan\PhpDocParser\Parser\TypeParser;
+
+class PHPStanDocblockParserFactory implements DocBlockFactory
```

Comment:
this would now be a _bridge_ to PHPStan, so the namespace for this code should be `...\Bridge\PHPStan\`

Reply:
moved to separate namespace, yet for now I left the (redundant) PHPStan prefix in class names (easier during testing and switching with original parser)